core/templates/domain/topics_and_skills_dashboard/topics-and-skills-dashboard-backend-api.service.spec.ts

```diff
@@ -13,98 +13,82 @@
 // limitations under the License.
 
 /**
- * @fileoverview Unit tests for CreatorDashboardBackendApiService.
+ * @fileoverview Unit tests for Topics and skills dashboard.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
 
-import { TranslatorProviderForTests } from 'tests/test.extras';
+import { HttpClientTestingModule, HttpTestingController } from
+  '@angular/common/http/testing';
+import { fakeAsync, flushMicrotasks, TestBed } from '@angular/core/testing';
+import { TopicsAndSkillsDashboardBackendApiService } from
+  // eslint-disable-next-line max-len
+  '../topics_and_skills_dashboard/topics-and-skills-dashboard-backend-api.service';
 
-require(
-  'domain/topics_and_skills_dashboard/' +
-  'topics-and-skills-dashboard-backend-api.service.ts');
-require('domain/utilities/url-interpolation.service.ts');
+describe('Topics and Skills Dashboard backend API service', () => {
+  let topicsAndSkillsDashboardBackendApiService:
+    TopicsAndSkillsDashboardBackendApiService = null;
+  let httpTestingController: HttpTestingController = null;
 
-describe('Topics and Skills Dashboard backend API service', function() {
-  var TopicsAndSkillsDashboardBackendApiService = null;
-  var $httpBackend = null;
-  var UrlInterpolationService = null;
-  var SAMPLE_TOPIC_ID = 'hyuy4GUlvTqJ';
-
-  var sampleDataResults = {
-    topic_summary_dicts: [{
-      id: SAMPLE_TOPIC_ID,
-      name: 'Sample Name',
-      language_code: 'en',
-      version: 1,
-      canonical_story_count: 3,
-      additional_story_count: 0,
-      uncategorized_skill_count: 3,
-      subtopic_count: 3,
-      topic_model_created_on: 1466178691847.67,
-      topic_model_last_updated: 1466178759209.839
-    }],
-    skill_summary_dicts: []
-  };
-
-  var TOPICS_AND_SKILLS_DASHBOARD_DATA_URL =
+  const SAMPLE_TOPIC_ID = 'hyuy4GUlvTqJ';
+  const TOPICS_AND_SKILLS_DASHBOARD_DATA_URL =
```

Comment:
When you have multi-line statements, always wrap in `(...)` and break after `(` (see style guide [here](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript)).